erizo/src/erizo/pipeline/HandlerManager.h

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright (c) 2016, Facebook, Inc.
```

Comment:
Correct me if I'm wrong, but didn't you come up with this class? We don't need to put the license here in that case

Reply:
you got it